src/content/containers/Root.js

```diff
@@ -77,8 +84,20 @@ class ProfileViewWhenReadyImpl extends PureComponent {
           </div>
         );
       }
-      case 'FATAL_ERROR':
-        return <div>{"Couldn't load the profile from the store."}</div>;
+      case 'FATAL_ERROR': {
+        const message = ERROR_MESSAGES[dataSource] || "Couldn't retrieve the profile.";
+        let additionalMessage = null;
+        if (view.error) {
+          additionalMessage = `Error was "${view.error}". Please look the Web Console to find more information.`;
```

Comment:
Most of the time we can see the full stack in the web console. Maybe I should run `console.error(view.error)` here, and write "The full stack has been written to the Web Console" ?